src/app/clusters/microwave-oven-control-server/microwave-oven-control-server.cpp

```diff
@@ -163,15 +290,30 @@ void Instance::HandleSetCookingParameters(HandlerContext & ctx, const Commands::
     uint8_t reqCookMode;
     uint32_t reqCookTime;
     uint8_t reqPowerSetting;
-    auto & cookMode     = req.cookMode;
-    auto & cookTime     = req.cookTime;
-    auto & powerSetting = req.powerSetting;
+    uint8_t reqWattSettingIndex;
+    uint8_t maxPower = kDefaultMaxPower;
+    uint8_t minPower = kDefaultMinPower;
+    bool reqStartAfterSetting;
+    auto & cookMode          = req.cookMode;
+    auto & cookTime          = req.cookTime;
+    auto & powerSetting      = req.powerSetting;
+    auto & wattSettingIndex  = req.wattSettingIndex;
+    auto & startAfterSetting = req.startAfterSetting;
```

Comment:
why are these references? any reason for these to not be const?

Reply:
I see the request is const. Code seems ok, except I wonder if there is significant readability gain from saying `startAfterSetting` instead of `req.startAfterSetting` to offset the overhead of knowing or searching up that the former just means the latter.

Reply:
the auto references here are convenient for coding, I think they are also understandable for the reader.